src/appmixer/azuredocumentintelligence/classifiers/ClassifyDocumentFromStream/ClassifyDocumentFromStream.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const DocumentIntelligence = require('@azure-rest/ai-document-intelligence').default;
+const { isUnexpected, getLongRunningPoller } = require('@azure-rest/ai-document-intelligence');
+
+module.exports = {
+
+    async receive(context) {
+
+        const { endpoint, apiKey } = context.config;
+        const { classifierId, fileId } = context.messages.in.content;
+
+        const client = DocumentIntelligence(endpoint, { key: apiKey });
+
+        let options;
+
+        const fileInfo = await context.getFileInfo(fileId);
+        const fileStream = await context.getFileReadStream(fileId);
+        options = {
```

Comment:
Can `options` be const?

```suggestion
        const fileInfo = await context.getFileInfo(fileId);
        const fileStream = await context.getFileReadStream(fileId);
        const options = {
```